internal/errs2/ignore.go

```diff
@@ -9,14 +9,19 @@ import (
 
 	"github.com/zeebo/errs"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
 // IsCanceled returns true, when the error is a cancellation.
 func IsCanceled(err error) bool {
 	return errs.IsFunc(err, func(err error) bool {
+		status, ok := status.FromError(err)
```

Comment:
this won't fail if `err` is not a grpc error, correct?

Reply:
Ok will be false in that case, hence `(ok && status.Code()...)`.

Reply:
@egonelbre could you review the changes in this function?

Reply:
@bryanchriswhite wouldn't `status.Code(err) == codes.Canceled` be a simpler check?

Reply:
It would seem so. This is the first time i've used `grpc/status` & `grpc/codes`.